.github/workflows/test.yml

```diff
@@ -80,33 +85,36 @@ jobs:
     strategy:
       matrix:
         include:
+          # Minimum supported rust (MSRV)
           - container:
-              image: georust/proj-ci:rust-1.53
+              image: georust/proj-ci:proj-9.0.0-rust-1.52
               env:
                 _PROJ_SYS_TEST_EXPECT_BUILD_FROM_SRC: 0
             features: ""
           - container:
-              image: georust/proj-ci:rust-1.52
```

Comment:
Sorry that this diff is confusing. It might be helpful to review the commits individually. The first commit updates the CI containers without updating proj, just to be sure we're using recent rust. 

It also re-orders the tasks in a way that I think makes the test matrix more clear.

The remaining two commits actually update to proj-9.0.0